### ISSUE TITLE
Strange categories list based on current keyboard language

### ISSUE THREAD

**Issue by PaulAnnekov** [mapsme/omim#3448]
1. Click menu item to add new POI to the map.
2. You will see a list of categories in language of your current keyboard language. Suppose it's english.
3. Switch to 2nd language of your keyboard. Suppose it's russian.
4. The list will still be in english.
5. Try to enter some text in search bar.
6. List will become russian.

Video: https://youtu.be/196_pDGSQy8

It looks very strange. Why list language depends on keyboard language and not system language? If it's your decision, then why you don't immediately switch list language on keyboard language switch?

**Comment by dbf256** [comment]
I agree, I updated to MAPS.ME android 6.1.8-Google and list of POI is in English now and I can't change it so it is very difficult to add new items. If all application menus are in Russian why to make single list in English?
